scrnaseq.Rmd

```diff
@@ -1997,7 +2151,7 @@ if (markers_found) {
   for(n in levels(marker_genesets_top_enriched$Cluster)){
     cat('##### ', n, ' \n')
     
-    cat(knitr::kable(marker_genesets_top_enriched %>% dplyr::ungroup() %>% dplyr::filter(Cluster==n) %>% dplyr::select(Database, Term, Direction, Adjusted.P.value, Odds.Ratio, Combined.Score), 
+    print(knitr::kable(marker_genesets_top_enriched %>% dplyr::ungroup() %>% dplyr::filter(Cluster==n) %>% dplyr::select(Database, Term, Direction, Adjusted.P.value, Odds.Ratio, Combined.Score), 
```

Comment:
I have to check whether multiple windows open again on the Mac...

Reply:
ok